railties/lib/rails/generators/rails/app/templates/config/databases/ibm_db.yml

```diff
@@ -11,14 +11,14 @@
 #       export IBM_DB_LIB=/opt/ibm/db2/V9.7/lib32
 #       gem install ibm_db
 #
-#   On Mac OS X 10.5:
+#   On macOS 10.5:
```

Comment:
Based on their [support document](https://support.apple.com/en-us/HT2405)  they still call OS X 10.5 as "Mac OS X 10.5". Should we keep them as-is for legacy version?